### PR TITLE
docs(zh): updated guide docs 950763f...a0b40f0

### PR DESCRIPTION
**Pull request by Jinjiang**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe: Chinese docs translation update

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch.
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing: https://github.com/vuejs/vue-test-utils/blob/dev/.github/CONTRIBUTING.md#development-setup
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Ref: 950763f...a0b40f0
/cc @Justineo

### CODE REVIEW

File: docs/zh/guides/getting-started.md [hunk 1]
```diff
@@ -104,30 +122,31 @@ it('button click should increment the count', () => {
 
 为了测试计数器中的文本是否已经更新，我们需要了解 `nextTick`。
 
-### 使用 `nextTick`
+### 使用 `nextTick` 和 await 行动
```
Comment (Justineo) [inline docs/zh/guides/getting-started.md]: 感觉“使用 `nextTick` 并 await 操作”或者“使用 `nextTick` 与 await”更通顺一些

File: docs/zh/guides/getting-started.md [hunk 2]
```diff
@@ -104,30 +122,31 @@ it('button click should increment the count', () => {
...
 
-Vue 会异步的将未生效的 DOM 批量更新，避免因数据反复变化而导致不必要的渲染。
+任何导致操作 DOM 的改变都应该在断言之前 await `nextTick` 函数。因为 Vue 会异步地将未生效的 DOM 批量_异步更新_，避免因数据反复变化而导致不必要的渲染。
```
Comment (Justineo) [inline docs/zh/guides/getting-started.md]: “Vue 会对未生效的 DOM 进行批量*异步更新*”（要用 *）

File: docs/zh/guides/getting-started.md [hunk 3]
```diff
@@ -104,30 +122,31 @@ it('button click should increment the count', () => {
...
 
 _你可以阅读[Vue 文档](https://cn.vuejs.org/v2/guide/reactivity.html#异步更新队列)了解更多关于异步指更新的信息。_
```
Comment (Justineo) [inline docs/zh/guides/getting-started.md]: 用*

File: docs/zh/guides/getting-started.md [hunk 4]
```diff
@@ -156,7 +175,9 @@ it('will catch the error using async/await', async () => {
 })
 ```
 
-### 下一步是什么
+### 接下来
+
 
-- [选择一个测试运行器](./choosing-a-test-runner.md)以把 Vue Test Utils 集成到你的工程里。
-- 移步[撰写测试的常见技巧](./common-tips.md)以学习更多。
+- 移步[撰写测试的常见技巧](./README.md#明白要测试的是什么)以学习更多。
```
Comment (Justineo) [inline docs/zh/guides/getting-started.md]: 我一直倾向于用“编写”而不是“撰写”，后者感觉有点过于正式了……

File: docs/zh/guides/testing-async-components.md [hunk 5]
```diff
@@ -11,17 +11,21 @@ Vue 会异步的将未生效的 DOM 批量更新，避免因数据反复变化
...
+it('button click should increment the count text', async () => {
+  expect(wrapper.text()).toContain('0')
+  const button = wrapper.find('button')
+  await button.trigger('click')
+  expect(wrapper.text()).toContain('1')
+})
+```
 
-// 其它的代码片断...
+Awaiting the trigger above is the same as doing:
```
Comment (Justineo) [inline docs/zh/guides/testing-async-components.md]: 漏翻了

File: docs/zh/guides/testing-async-components.md [hunk 6]
```diff
@@ -31,11 +35,20 @@ it('button click should increment the count text', async () => {
 })
 ```
 
+Methods that can be awaited are:
```
Comment (Justineo) [inline docs/zh/guides/testing-async-components.md]: 同上

File: docs/zh/guides/testing-async-components.md [hunk 7]
```diff
@@ -113,8 +128,14 @@ it('fetches async when a button is clicked', async () => {
   const wrapper = shallowMount(Foo)
   wrapper.find('button').trigger('click')
   await flushPromises()
-  expect(wrapper.vm.value).toBe('value')
+  expect(wrapper.text()).toBe('value')
 })
 ```
 
 相同的技术细节也可以应用在处理 Vue Actions 上，默认情况下，它也会返回一个 Promise。
```
Comment (Justineo) [inline docs/zh/guides/testing-async-components.md]: Vuex 的 action

File: docs/zh/guides/testing-async-components.md [hunk 8]
```diff
@@ -113,8 +128,14 @@ it('fetches async when a button is clicked', async () => {
...
+
+#### 为什么不使用 `await button.trigger()`？
+
+如之前所解释的，Vue 更新其组件的时机会不同，这个时间被带入了一个 Promise 对象，如 `axios` 解析出的那个。
```
Comment (Justineo) [inline docs/zh/guides/testing-async-components.md]: 不同指的是“更新组件”vs“普通的 Promise”，原文用的是“between”，是前后半句内容之间有差异

File: docs/zh/guides/usage-without-a-build-step-node.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
+## 不经过构建而使用
+
+在我们习惯于使用工具，诸如 [Webpack](https://webpack.js.org/) 打包 Vue 应用、`vue-loader` 处理单文件组件、[Jest](https://jestjs.io/) 转写富于表现力测试的同时，使用 Vue Test Utils 其实不需要引入这么多。除了本库之外，使用 Vue Test Utils 最低要求是：
```
Comment (Justineo) [inline docs/zh/guides/usage-without-a-build-step-node.md]: webpack，Vue Loader

File: docs/zh/guides/useful-libraries-for-testing.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+## 实用的测试库
+
+Vue Test Utils 提供了测试 Vue 组件的实用方法。同时我们的社区也有一些额外的库为 `@vue/test-utils` 拓展了更多实用的方法，或提供测试 Vue 应用中其它内容的工具。
+
+### `vuex-mock-store`
+
+[`vuex-mock-store`](https://github.com/posva/vuex-mock-store) 提供了一个简单直接的伪造 store 来简化消费 Vuex store 的组件的测试。
```
Comment (Justineo) [inline docs/zh/guides/useful-libraries-for-testing.md]: 使用 store 吧

File: docs/zh/guides/usage-without-a-build-step-node.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    <div>{{ msg }}</div>
+  `
+})
+
+const wrapper = VueTestUtils.shallowMount(App)
+
+assert.strictEqual('Hello Vue Test Utils', wrapper.text())
+```
+
+如你所见的一样简单。不过因为我们没有构建步骤，我们无法使用单文件组件。且没有什么可以阻止我们通过 `<script>` 标签从 CDN 引入并以同样的方式使用 Vue。
```
Comment (Justineo) [inline docs/zh/guides/usage-without-a-build-step-node.md]: 且改成但更通一些

File: docs/zh/guides/using-with-vuex.md [hunk 12]
```diff
@@ -375,6 +377,14 @@ test('updates "evenOrOdd" getter when "increment" is committed', () => {
 
 注意我们在创建一个 store 之前使用了 `cloneDeep` 来克隆 store 配置。这是因为 Vuex 会改变用来创建 store 的选项对象。为了确保我们能为每一个测试都提供一个干净的 store，我们需要克隆 `storeConfig` 对象。
 
+然而，`cloneDeep` 不足以 `deep` 到克隆 store 的模块。如果你的 `storeConfig` 包含模块，你需要向 `new Vue.Store()` 传入一个对象，例如：
```
Comment (Justineo) [inline docs/zh/guides/using-with-vuex.md]: “deep”

**Review by Justineo — CHANGES_REQUESTED** [review]

File: docs/zh/guides/useful-libraries-for-testing.md (hunk 10 above)
Comment (Jinjiang) [reply docs/zh/guides/useful-libraries-for-testing.md]: 你的意思是“消费”->“使用”？

File: docs/zh/guides/testing-async-components.md [hunk 13]
```diff
@@ -132,10 +132,10 @@ it('fetches async when a button is clicked', async () => {
 })
 ```
 
-相同的技术细节也可以应用在处理 Vue Actions 上，默认情况下，它也会返回一个 Promise。
+相同的技术细节也可以应用在处理 Vuex 的 action 上，默认情况下，它也会返回一个 Promise。
 
 #### 为什么不使用 `await button.trigger()`？
 
-如之前所解释的，Vue 更新其组件的时机会不同，这个时间被带入了一个 Promise 对象，如 `axios` 解析出的那个。
+如之前所解释的，Vue 更新其组件和带入一个普通 Promise 对象的时机不同，如 `axios` 解析出的那个。
```
Comment (Justineo) [inline docs/zh/guides/testing-async-components.md]: > As explained above, there is a difference between the time it takes for Vue to update its components, and the time it takes for a Promise, like the one from axios to resolve.

应该是“Vue 更新其组件花的时间”vs.“一个 Promise (完成) 所花的时间”

File: docs/zh/guides/useful-libraries-for-testing.md (hunk 10 above)
Comment (Justineo) [reply docs/zh/guides/useful-libraries-for-testing.md]: 对

**Review by Justineo — APPROVED** [review]

**Pull request merged**